core/extensions.js

```diff
@@ -28,7 +29,7 @@ goog.requireType('Blockly.Block');
  * The set of all registered extensions, keyed by extension name/id.
  * @private
  */
-Blockly.Extensions.ALL_ = Object.create(null);
+const ALL = Object.create(null);
```

Comment:
This object is not "deeply immutable" (it gets used as a mutable dictionary) so the variable should not be named as if it were a constant.  `allExtensions` would probably better name.

Also—as has been discussed at length recently—declaring this `@private` is disallowed by the style guide (though I'm fine with leaving it for now, until we have consensus about the issue).

Reply:
Thanks for pointing that out.
I'll update the name so it's no longer using CONSTANT_CASE, which is supposed to be reserved for true "constants".